search_gov_crawler/search_gov_spiders/pipelines.py

```diff
@@ -19,30 +18,31 @@ def __init__(self, *_args, **_kwargs):
         self.current_file_size = 0
         self.file_number = 1
         self.parent_file_path = Path(__file__).parent.parent.resolve()
-        self.base_path_name = str(self.parent_file_path / "output/all-links.csv")
-        self.short_file = open(self.base_path_name, "w", encoding="utf-8")
-        self.max_file_size = 3900
+        self.base_path_name = str(self.parent_file_path / f"output/all-links-p{os.getpid()}.csv")
```

Comment:
i guess pid+YYYYMMDD might be more unique since they are technically reused but pid by iteself will probably do.  I guess our hope is not to really use the file writing option too often.

Reply:
I'm not sure I understand what you mean by they are technically reused?

Reply:
Ah you mean not in the same run? I didn't assume these were being saved week to week here. All the ones in drive are overwritten every week